tracing-core/src/lib.rs

```diff
@@ -219,6 +216,14 @@ pub mod span;
 pub(crate) mod stdlib;
 pub mod subscriber;
 
+// Vendored version of spin 0.5.2 (0387621)
+// `mutex` and `once` modules only
```

Comment:
Ideally, I'd really like it if this comment also had a note explaining *why* it's necessary to do this.